<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="first_player">
    <attribute name="title">
     <string>First Player</string>
    </attribute>
    <widget class="QLabel" name="img_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_1">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>250</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QComboBox" name="chose_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Москва</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лондон</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Париж</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Тула</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="res_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> Твой результат</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="second_player">
    <attribute name="title">
     <string>Two player</string>
    </attribute>
    <widget class="QComboBox" name="chose_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Москва</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лондон</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Париж</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Тула</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spin_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QLabel" name="img_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="res_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>253</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Твой результат</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> Отправить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
